### PR TITLE
Add Editorconfig-Checker

### PR DESCRIPTION
**Pull request by mstruebing**
I've added https://github.com/editorconfig-checker/editorconfig-checker.php to the list of code-analysis tools.

Let me know if I did something wrong. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -355,6 +355,7 @@ Please see [CONTRIBUTING](https://github.com/ziadoz/awesome-php/blob/master/CONT
 
 * [Better Reflection](https://github.com/Roave/BetterReflection) - AST-based reflection library that allows analysis and manipulation of code
 * [Code Climate](https://codeclimate.com) - An automated code review.
+* [Editorconfig-Checker](https://github.com/editorconfig-checker/editorconfig-checker.php) - A command line utility which verifies if your files implement your `.editorconfig` rules.
```
Comment (dereuromark) [inline README.md:358]: if => that

Comment (dereuromark) [reply README.md:358]: Also, maybe use https://github.com/editorconfig-checker as link?

Comment (mstruebing) [reply README.md]: Fixed.

**Comment by dereuromark** [comment]
Still a conflict though

**Review by dereuromark — APPROVED** [review]

**Pull request merged**